PusherMainView/PusherMainView/PusherViewController.swift

```diff
@@ -105,3 +107,15 @@ extension PusherViewController: PusherInteractable {
         apnsAuthTokenRadioButton.state = state.authTokenRadioState
     }
 }
+
+// MARK: - NSTextFieldDelegate
+
+extension PusherViewController: NSTextFieldDelegate {
+    public func controlTextDidChange(_ notification: Notification) {
```

Comment:
It has to be marked as public:
```
Method 'controlTextDidChange' must be declared public because it matches a requirement in public protocol 'NSControlTextEditingDelegate'
```